web_apps_src/web-element/package.json

```diff
@@ -22,7 +22,7 @@
     "generate": "stencil generate"
   },
   "dependencies": {
-    "@identifo/identifo-auth-js": "file:../identifo.js",
+    "@identifo/identifo-auth-js": "^3.1.3",
```

Comment:
I believe we need to use the local version if w rebuild it from here and the global version if we build it as an independent project. @sokolovstas